src/common/crypto/mbedtls/cryptoprovider.cpp

```diff
@@ -15,6 +18,85 @@
 namespace aos {
 namespace crypto {
 
+/***********************************************************************************************************************
+ * Helpers
+ **********************************************************************************************************************/
```

Comment:
change header to static

Reply:
+

Reply:
basically static is a mech to make functions local
I prefer naming based on their purpose.
Leaved it as is.

Reply:
changed to a uniform name